bootstrap/sql/com.mysql.cj.jdbc.Driver/v008__create_db_connection_info.sql

```diff
@@ -45,7 +45,7 @@ SET json = JSON_INSERT(
 )
 WHERE name = 'columnValuesToBeBetween';
 
-UPDATE pipeline_entity 
+UPDATE pipeline_entity
 SET json = JSON_INSERT(
```

Comment:
Why is this change required? Note we can never change previously release SQL files. Otherwise Flyway bootstrap will have problems.